### PR TITLE
App details

### PR DESCRIPTION
**Pull request by mayankamencherla**
Added getter and setter methods for app details

### CODE REVIEW

File: razorpay/client.py [hunk 1]
```diff
@@ -80,6 +82,12 @@ def _get_version(self):
             pass
         return version
 
+    def _set_app_details(self, app_details):
+        self.app_details.append(app_details)
+
+    def _get_app_details(self):
```
Comment (gaurav1068) [inline razorpay/client.py]: app_details are not getting used anywhere, refer to this
https://github.com/razorpay/razorpay-php/blob/master/src/Request.php#L157

Comment (mayankamencherla) [inline razorpay/client.py]: @gaurav1068 Added.

File: razorpay/client.py [hunk 2]
```diff
@@ -80,6 +83,26 @@ def _get_version(self):
             pass
         return version
 
+    def _get_app_details_ua(self):
+        appDetailsUa = ""
+
+        app_details = self._get_app_details()
```
Comment (gaurav1068) [inline razorpay/client.py]: _get_app_details fn is not required(it is private by def), if you want to create a public function remove starting _

File: razorpay/client.py [hunk 3]
```diff
@@ -80,6 +83,26 @@ def _get_version(self):
             pass
         return version
 
+    def _get_app_details_ua(self):
+        appDetailsUa = ""
```
Comment (gaurav1068) [inline razorpay/client.py]: underscore notation instead of camel case

File: razorpay/client.py [hunk 4]
```diff
@@ -80,6 +83,26 @@ def _get_version(self):
...
+
+        app_details = self._get_app_details()
+
+        for app in app_details:
+            if 'title' in app and isinstance(app['title'], str):
```
Comment (gaurav1068) [inline razorpay/client.py]: here you are checking for str, and below not list, make it consistent check for string at both the place

File: razorpay/client.py [hunk 5]
```diff
@@ -80,6 +83,26 @@ def _get_version(self):
...
+                appUa = app['title']
+                if 'version' in app and not isinstance(app['version'], list):
+                    appUa += "{}{}".format('/', app['version'])
```
Comment (gaurav1068) [inline razorpay/client.py]: appUa += "{}{}".format('/', app['version'])
to 
app_ua += "/{}".format(app['version'])

Comment (mayankamencherla) [inline razorpay/client.py]: Changed.

File: razorpay/client.py (hunk 4 above)
Comment (mayankamencherla) [inline razorpay/client.py]: Changed.

File: razorpay/client.py (hunk 2 above)
Comment (mayankamencherla) [inline razorpay/client.py]: Made public

File: razorpay/client.py (hunk 3 above)
Comment (mayankamencherla) [inline razorpay/client.py]: Done

File: razorpay/client.py [hunk 6]
```diff
@@ -1,4 +1,5 @@
 import os
+import sys
```
Comment (gaurav1068) [inline razorpay/client.py]: unused remove this

**Pull request merged**